### ISSUE TITLE
Latest release is broken

### ISSUE THREAD

**Issue by dignifiedquire** [mxinden/if-watch#20]
`1.1.0` and current `master` are broken on linux (ubuntu) for me (rustc 1.16.0)

```
> cargo run --example if_watch
error[E0432]: unresolved import `netlink_proto::ErrorKind`
  --> /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/rtnetlink-0.9.1/src/lib.rs:47:9
   |
47 |         ErrorKind,
   |         ^^^^^^^^^ no `ErrorKind` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `rtnetlink` due to previous error
```

**Comment by dignifiedquire** [comment]
Fixed by #19

**Comment by mxinden** [comment]
#19 is merged and published. Thanks @dignifiedquire for reporting!
